sts/security-sts-server/src/main/java/ddf/security/sts/PropertyPlaceholderWrapper.java

```diff
@@ -45,10 +43,15 @@ public PropertyPlaceholderWrapper(DefaultConditionsProvider conditionsProvider,
             StaticSTSProperties properties) {
         samlConditionsProvider = conditionsProvider;
         stsProperties = properties;
+        // set the default values in case there is no configuration
+        samlConditionsProvider.setLifetime(1800);
```

Comment:
Just curious, would it be worthwhile to check if there are already values in the samlConditionsProvider and stsProperties before setting them to default values? Not sure if that's something that could happen (i.e. values passed in to the constructor).